src/compiler/utilities.ts

```diff
@@ -6302,7 +6298,7 @@ export function getPossibleOriginalInputExtensionForExtension(path: string) {
 
 /** @internal */
 export function outFile(options: CompilerOptions) {
-    return options.outFile || options.out;
+    return options.outFile;

```

Comment:
Can we remove this helper and use `compilerOptions.outFile` directly everywhere now?

Reply:
Sure, I can inline this (and the other one).

Reply:
Done.